src/main/java/ch/njol/skript/classes/data/DefaultChangers.java

```diff
@@ -349,5 +352,56 @@ public void change(final Block[] blocks, final @Nullable Object[] delta, final C
 			}
 		}
 	};
+	public final static Changer<BossBar> bossbarChanger = new Changer<BossBar>() {
+		
+		@Nullable
+		@Override
+		public Class<?>[] acceptChange(ChangeMode mode) {
+			switch (mode) {
+				case REMOVE:
+				case DELETE:
+				case ADD:
+					return new Class[]{Player.class};
+				default:
+					return null;
+			}
+		}
+		
+		@Override
+		public void change(BossBar[] what, @Nullable Object[] delta, ChangeMode mode) {
+			switch (mode) {
+				case DELETE:
+					for (BossBar bossBar : what) {
+						if (bossBar instanceof KeyedBossBar) {
+							bossBar.removeAll();
+							bossBar.setVisible(false);
+							Bukkit.removeBossBar(((KeyedBossBar) bossBar).getKey());
+						} else {
+							bossBar.removeAll();
+							bossBar.setVisible(false);
+						}
+					}
+					break;
+				case ADD:
+					if (delta != null) {
```

Comment:
delta cannot be null here, check redundant, same goes for a lot of other change methods:
delta is only null for DELETE or RESET changer